Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -340,13 +340,15 @@ final class ManagePledgeViewController: UIViewController, MessageBannerViewContr
   // MARK: - Functions
 
   private func goToRewards(_ project: Project) {
-    let rewardsVC = RewardsCollectionViewController.instantiate(
+    let vc = RewardsCollectionViewController.instantiate(
       with: project,
       refTag: nil,
       context: .managePledge
     )
 
-    self.navigationController?.pushViewController(rewardsVC, animated: true)
+    vc.pledgeViewDelegate = self
+
+    self.show(vc, sender: nil)
```

Comment:
Any reason to use `show` instead of our regular `pushVC`? From the docs it seems that the `show` function is only really useful if you're using something like `UISplitViewController`, and elsewhere in the app we tend to use `push`. 🤔

> You use this method to decouple the need to display a view controller from the process of actually presenting that view controller onscreen. Using this method, a view controller does not need to know whether it is embedded inside a navigation controller or split-view controller. It calls the same method for both.

 Seems to me like it might be breaking with what we're using elsewhere without a clear benefit, but I might be missing something.

Reply:
I noticed it was used in a couple other places in this VC but yeah it seems it's only really useful if we are using `UISplitViewController` to abstract away which control the VC is embedded in. I'll revert to push here.